wifiutils/src/main/java/com/thanosfisherman/wifiutils/ConfigSecurities.java

```diff
@@ -22,7 +22,7 @@
 import static com.thanosfisherman.wifiutils.WifiUtils.wifiLog;
 import static com.thanosfisherman.wifiutils.utils.SSIDUtils.convertToQuotedString;
 
-final class ConfigSecurities {
+final public class ConfigSecurities {
```

Comment:
Is there any particular reason why we should expose all these methods in `ConfigSecurities` as public?